packages/twenty-front/src/modules/ui/navigation/link/components/AdvancedSettingsToggle.tsx

```diff
@@ -46,13 +48,17 @@ export const AdvancedSettingsToggle = () => {
     setIsAdvancedModeEnabled(newValue);
   };
 
+  const toggleAdvancedMode = () => {
+    setIsAdvancedModeEnabled((current) => !current);
+  };
+
   return (
     <StyledContainer>
       <StyledIconContainer>
         <StyledIconTool size={12} color={MAIN_COLORS.yellow} />
       </StyledIconContainer>
       <StyledToggleContainer>
```

Comment:
I think we should make the Container clickable
@Devessier what's your recommendation here ? use a button?

I feel the "Advanced" is actually a label for the toggle

Reply:
Thanks for mentioning me, @charlesBochet.

Indeed, I think we should make the text "Advanced:" a true HTML `<label>` element. I don't believe the whole container should be clickable, as clicking on a blank space would update the toggle, which might seem strange.
The issue is that the `<Toggle />` component is only made of `<div>` and not a proper `<input>` element, so using a `<label>` would do nothing. I had this component on my list of accessibility things to discuss.

See https://github.com/twentyhq/twenty/blob/05e8f8a0b1fd4c0b1e53c289baba3d073565b61a/packages/twenty-front/src/modules/ui/input/components/Toggle.tsx

Reply:
@Vardhaman619 could you update our Toggle component to include a <input>
Take inspiration from chakra ui regarding the html structure
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/1d0f6510-27d0-4b9e-9b46-5975532fa09b">

It looks to me that they have an invisible input

Reply:
Please note that the styles applied to hide the `input` are to make it visible only for screen readers (the tool used by people with disabilities).

These styles are now normalized and everybody uses the same to hide an element for people not using a screen reader:

```css
// In Tailwind:

.sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  white-space: nowrap;
  border-width: 0;
}
```

Some resources : 

- https://webaim.org/techniques/css/invisiblecontent/
- https://tailwindcss.com/docs/screen-readers

Reply:
We can introduce a <InvisibleInput /> component in this case, let's put it in an accessibility folder in twenty-ui

Reply:
Chakra UI has two components : VisibilityHidden and VisibilityHiddenInput. (https://v2.chakra-ui.com/docs/components/visually-hidden/usage#visually-hidden-input)

Their VisibilityHidden component renders a `<span>` as it's usually used to provide little indications to the users, like a hidden label explaining what shows an icon.